src/indra_cogex/apps/queries_web/__init__.py

```diff
@@ -466,7 +472,8 @@ def get_example_data():
     [(subnetwork, fn) for fn in ["indra_subnetwork_relations", "indra_subnetwork_meta", "indra_mediated_subnetwork",
                                  "indra_subnetwork_tissue", "indra_subnetwork_go"]] +
     [(metabolite_analysis, fn) for fn in ["metabolite_discrete_analysis"]] +
-    [(gene_analysis, fn) for fn in ["discrete_analysis", "signed_analysis", "continuous_analysis"]]
+    [(gene_analysis, fn) for fn in ["discrete_analysis", "signed_analysis", "continuous_analysis"]] +
+    [(search, fn) for fn in ["get_ora_statements"]]
```

Comment:
When I test `get_ora_statements` in the rest API docs, it errors. Can you look into the examples there? We might have to add a special case for `get_ora_statements`